src/admin/adminDashboard.tsx

```diff
@@ -0,0 +1,27 @@
+import Button from '@mui/material/Button';
+import React from 'react'
+import { Link } from 'react-router-dom';
+import { useStateValue } from '../context/StateProvider';
+import AdminPageContainer from './adminPageContainer'
+
+function AdminDashboard() {
+    const [{ userInfo }] = useStateValue();
+
+    return (
+        <AdminPageContainer userInfo={userInfo}>
+            <div>
+                <Link to="/admin/poster/?JohnLennonTrippy">
```

Comment:
should this be a list?

Reply:
Yes, planning to do it later. Just want to keep the PR smaller, and do a new feature to make a proper dashboard